src/pages/patientView/clinicalInformation/PatientViewPageStore.spec.ts

```diff
@@ -35,4 +35,20 @@ describe('PatientViewPageStore', () => {
         assert.deepEqual(result,[]);
     });
 
+
+    it.only('sets page title to patient if theres a patient id and sample if sample id, patient id winning out', ()=>{
+
+        assert.equal(store.pageTitle, 'Patient: ');
+
+        store.setPatientId('1234');
+        assert.equal(store.pageTitle, 'Patient: 1234');
+
+        store.setSampleId('1234');
+        assert.equal(store.pageTitle, 'Sample: 1234');
```

Comment:
aren't there both patient and sample ids at this point, shouldn't patient still win out?